UI-extra/style.css

```diff
@@ -540,12 +520,31 @@ p {
     margin-left:60px;
 }
 
-.mini-cart-bundle:before {
+.bullet-item:before {
```

Comment:
Let's try and keep `.bullet-item` styles near to each other so its clear to see what's happening.